src/ray/rpc/retryable_grpc_client.h

```diff
@@ -222,6 +223,9 @@ class RetryableGrpcClient : public std::enable_shared_from_this<RetryableGrpcCli
       pending_requests_;
   // Total number of bytes of pending requests.
   size_t pending_requests_bytes_ = 0;
+  // TODO(57156): this is messy to leave in the retryable grpc client, refactor this
+  // Total number of inflight requests.
+  std::atomic<size_t> num_active_requests_ = 0;
```

Comment:
we should holistically revisit metrics/counters across the retryable grpc and regular grpc client